gitbook-docs/market/cmd-reference/statement.md

```diff
@@ -1,75 +1,104 @@
 ---
-description: Scripts to manage statements/circuits
+description: Managing circuits and statements
 ---
 
-# statement
+# Statement utilities
 
+This page describes scripts for circuit developers and proof producers,
+`prepare_statement.py` and `statement_tools.py`.
 
+Make sure you've [signed up](user.md) before continuing.
 
-### Prepare Statement
+## Preparing circuit as a statement
 
-Statements/circuits generated by the zkLLVM compiler need to have additional metadata attached to them before they can be published on the proof market.
+Circuits are stored as a *statement* structure on the Proof Market.
+Statement examples can be found in `./example/statements/` directory.
+Attach all additional metadata to circuits generated by the zkLLVM compiler
+before publishing them on the Proof Market.
```

Comment:
Looks like this should be in the Examples section

Reply:
Thanks, moved it